Core/src/Visualization/GeometryView3D.cpp

```diff
@@ -191,42 +192,23 @@ void Acts::GeometryView3D::drawPortal(IVisualization3D& helper,
 
 void Acts::GeometryView3D::drawDetectorVolume(
     IVisualization3D& helper, const Experimental::DetectorVolume& volume,
-    const GeometryContext& gctx, bool drawSurfaces, const Transform3& transform,
```

Comment:
So this boolean switch just has no use case and is therefore removed?

Reply:
yeah exactly. I think the surfaces should always be a part of the volume. whenever we want to have that optional we can add this back.